corona-warn-app/cwa-app-android
# Check-In Personal setting adjustment (EXPOSUREAPP-6335)

This PR address Exposureapp 6335.
(Slide state must be safed after toggeling)

**How to test:**

- Tab Bar > Check-in
- check in to one event and swipe off the automatic contact journal creation 
- check in to another event and check the slider state of the automatic contact journal creation (must be identical)

**Screenshot:**
![image](https://user-images.githubusercontent.com/46747780/115051812-8322f780-9edd-11eb-906d-8c1c751632bc.png)
